src/app/(root)/(auth)/login/page.tsx

```
@@ -0,0 +1,62 @@
+          <Image src="/image/logo_gray.webp" alt="login_logo" width={100} height={90} priority />
+        </div>
+
+        <div className="flex flex-col space-y-1.25 text-center typo-body-m-1">
+          <button
+            onClick={handleKakaoLogin}
+            className={`
+              flex h-15 w-full cursor-pointer items-center justify-center
+              rounded-full bg-kakao p-0 text-black
+            `}
+          >
+            카카오로 로그인 하기
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)